src/LinkedList/linked_list.c

```diff
@@ -28,6 +28,10 @@ pLinkedList makeList() {
   // Allocate enough space for a linked list, but malloc return a (void*)
   // We need to cast it to a pLinkedList.
   pLinkedList linkedList = (pLinkedList) malloc(sizeof(LinkedList));
+  if (linkedList == NULL) {
+    printf("Malloc failed\n");
```

Comment:
We don't want to print anything here. Let the client decided what to do. What do you think? (This apply elsewhere).